src/WebGL/Renderer.ts

```diff
@@ -69,6 +69,7 @@ export class Renderer {
   // GL state cache
   private _oldDoubleSided = -1 as number | boolean;
   private _oldFlipSided = -1 as number | boolean;
+  // @ts-ignore
   private _oldBlending = -1;
```

Comment:
Regarding, https://github.com/3dmol/3Dmol.js/pull/712#discussion_r1303000686, I wasnt sure if I should go ahead and remove unused variables in those places. Hence added // @ts-ignore. We can discuss it on our call today and finish this one up :)

Reply:
Removed the unused variables and `// @ts-ignore` :)